docs/platform_management_plan/project_management.rst

```diff
@@ -0,0 +1,165 @@
+* documentation of the :ref:`software development process <Process_Description>`
+* build system and toolchain definition
+* integration rules for software modules.
+
+The main repository references multiple other repositories, mostly repositories, where
+software modules or toolchains are defined. This results in the following :ref:`Folder Structure of Platform Repository <Platform_Folder_Structure>`. Every software module has its own repository, that contains multiple components, their requirements, architecture, implementation and tests.
+A software module and its repository can be part of the main SCORE *Eclipse Project* and corresponding *GitHub organization* or can be moved to a standalone *Eclipse child project*, if necessary.
+
+  .. image:: _assets/project_organization.svg
+     :width: 900
+     :alt: Infrastructure overview
+     :align: center
+
+Platform organization
+=======================
+Also in case the software module repositories are not always placed
+in standalone *Eclipse child projects*, we still consider all software modules
+to be standalone *Eclipse child projects*, having their own *Committers* and *Project Leads*
+as defined by the *Eclipse Foundation Project Handbook*. Software module committers
+and software module project leads are responsible for managing the software module as if it were
+a normal *Eclipse child project*. The election of the project leads and committers for software module projects should be done using the main integration *SCORE* project mailing list, *score-dev@eclipse.org*. This means, that the decision who will be the project lead and committer of the new software module will be taken by the project leads and committers of the main *SCORE* project respectively. The elected project leads or committers of the software modules are not automatically project leads and committers of the main integration *SCORE* project. Typically, before becoming a project lead or a committer of the main integration *SCORE* project, you need to build up a good reputation by contributing to the main integration *SCORE* project and being project lead or committer for one of the software modules.
+
+That should be a normal procedure, that before introducing a new *Eclipse child project* for a software module, it should first reside as a repository in the main *SCORE* project. If the software module later would be moved to a real standalone *Eclipse child project*, e.g., as there is a wish to use this software module independent of the *SCORE* project, then the elected project leads and committers of the software module will be simply taken over as project leads and committers of the new *Eclipse child project* and their tasks will stay the same. Further in this document differentation between a software module and  *Eclipse child project* will be done only if necessary. For the software module that resides in the separate repository of the main *SCORE* project, the configuration and the control
```

Comment:
The control over who is project leader and committer of projects can only be decided by the independent child project itself. I would propose to have only a rule about what independent projects have to fulfill when they want to be module inside the main score project. This can also be the case if the score projects decides to have a current independent projects as part of the modules without beeing anytime part of score repositories.